Tools/peg_generator/peg_parser/pegen.h

```diff
@@ -81,6 +88,9 @@ typedef struct {
     int is_keyword;
 } KeywordOrStarred;
 
+extern const int n_keyword_lists;
+extern KeywordToken *reserved_keywords[];
```

Comment:
I think another possibility is somehow to write the keywords on some header file that we include only on `pegen.h`. In that way, we don't need this dance with the `extern`. This means that the `c_generator` could generate a `.c` file *and* a header file. This also can help to inform other files to what is being exposed by `parse.c`. What do you think @lysnikolaou @gvanrossum ?

Reply:
What about [this idea](https://github.com/we-like-parsers/cpython/pull/11#discussion_r396182718) from @gvanrossum?

Reply:
> What about this idea from @gvanrossum?

Oh, missed that comment. Yeah, I like that idea!

Reply:
Hm, I think having two output files from the generator is a bit of a pain -- I think it has complicated the old "pgen" solutions a bit, so I'd like to avoid that this time around.